src/java.rmi/share/classes/java/rmi/activation/ActivationGroupDesc.java

```diff
@@ -308,11 +308,15 @@ public int hashCode()
          *
          * <p>This method reads this object's serialized form for this
          * class as follows:
-         *
          * <p>This method first invokes <code>defaultReadObject</code> on
          * the specified object input stream, and if <code>options</code>
          * is <code>null</code>, then <code>options</code> is set to a
          * zero-length array of <code>String</code>.
+         *
+         * @param  in the {@code ObjectInputStream} from which data is read
+         * @throws IOException if an I/O error occurs
+         * @throws ClassNotFoundException if a serialized class cannot be loaded
+         *
          */
         private void readObject(ObjectInputStream in)
```

Comment:
Hi Roger, you could take this opportunity to add the @serial annotation to this method (here and in the other files you modified as well.